src/Sensor_TEST.cc

```diff
@@ -24,12 +24,17 @@ using namespace sensors;
 
 class TestSensor : public Sensor
 {
-  public: bool Update(const common::Time &) override
+  public: bool Update(const std::chrono::steady_clock::duration &) override
   {
     updateCount++;
     return true;
   }
 
+  public: bool IGN_DEPRECATED(4) Update(const common::Time &) override
```

Comment:
The tests are not installed and used by downstream users, so they don't need to go through a deprecation cycle